app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -87,10 +87,12 @@ class PoolResource < JSONAPI::Resource
       #   @return [Float] the used volume of the pool
       # @!attribute [rw] available_volume
       #   @return [Float] the available volume of the pool
+      # @!attribute [r] barcode
+      #   @return [String] the barcode of the pool (via tube)
       attributes :volume, :concentration, :template_prep_kit_box_barcode,
                  :insert_size, :created_at, :updated_at,
                  :library_attributes, :used_aliquots_attributes, :primary_aliquot_attributes,
-                 :used_volume, :available_volume
+                 :used_volume, :available_volume, :barcode
```

Comment:
we have this as r but it is not marked as ready only. Is this an issue?

Reply:
Updated it [027bc54](https://github.com/sanger/traction-service/pull/1587/commits/027bc544038ef93648319046311cf8d9dfda4e98)